wdl/transforms/cascades/src/test/scala/wdl/transforms/cascades/linking/expression/values/CascadesValueEvaluatorSpec.scala

```diff
@@ -0,0 +1,189 @@
+package wdl.transforms.cascades.linking.expression.values
+
+import common.assertion.CromwellTimeoutSpec
+import common.assertion.ErrorOrAssertions._
+import common.assertion.ManyTimes.intWithTimes
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import wdl.model.draft3.elements.ExpressionElement
+import wdl.model.draft3.graph.expression.EvaluatedValue
+import wdl.model.draft3.graph.expression.ValueEvaluator.ops._
+import wdl.transforms.cascades.Ast2WdlomSpec.{fromString, parser}
+import wdl.transforms.cascades.ast2wdlom._
+import wom.expression.NoIoFunctionSet
+import wom.types.{WomIntegerType, WomMapType, WomOptionalType, WomStringType}
+import wom.values.{WomArray, WomInteger, WomMap, WomOptionalValue, WomPair, WomString}
+
+class CascadesValueEvaluatorSpec extends AnyFlatSpec with CromwellTimeoutSpec with Matchers {
+
+  behavior of "biscayne value evaluator"
+
+  // The "did we bring in the base transforms" sanity check:
+  it should "return a static integer from static integer addition" in {
+    val str = "3 + 3"
+    val expr = fromString[ExpressionElement](str, parser.parse_e)
+
+    expr.shouldBeValidPF {
+      case e => e.evaluateValue(Map.empty, NoIoFunctionSet, None) shouldBeValid EvaluatedValue(WomInteger(6), Seq.empty)
+    }
+  }
+
+  it should "evaluate an 'as_map' expression correctly" in {
+    val str = """ as_map( [("x", 1), ("y", 2), ("z", 3)] ) """
+    val expr = fromString[ExpressionElement](str, parser.parse_e)
+
+    val expectedMap: WomMap = WomMap(Map (
+      WomString("x") -> WomInteger(1),
+      WomString("y") -> WomInteger(2),
+      WomString("z") -> WomInteger(3)
+    ))
+
+    expr.shouldBeValidPF {
+      case e => e.evaluateValue(Map.empty, NoIoFunctionSet, None) shouldBeValid EvaluatedValue(expectedMap, Seq.empty)
+    }
+  }
+
+  it should "evaluate an 'as_pairs' expression correctly" in {
+    // Repeat this a few times ensure we aren't occasionally reordering by accident:
+    10 times {
+      val str = """ as_pairs( { 1: "one", 2: "two", 3: three } ) """
```

Comment:
Just confirming it's intentional that `three` is not in quotes.

Reply:
I wasn't able to find any evidence for this in either the 1.1 or development specs, so assuming typo. Test passes fine with quotes, so checking that in.

Reply:
Is this assuming that `three` is a WDL variable, and checking that the as_pairs function is correctly evaluating sub-clauses correctly?